doc/man1/flux-mini.adoc

```diff
@@ -0,0 +1,147 @@
+// flux-help-include: true
+FLUX-MINI(1)
+============
+:doctype: manpage
+
+
+NAME
+----
+flux-mini - Minimal Job Submission Tool
+
+
+SYNOPSIS
+--------
+*flux* *mini* *submit* [OPTIONS] ['--nprocs=N'] COMMAND...
+
+*flux* *mini* *run* [OPTIONS] ['--nprocs=N'] COMMAND...
+
+
+DESCRIPTION
+-----------
+flux-mini(1) submits jobs to run under Flux.  The job consists of
+'N' copies of COMMAND launched together as a parallel job.
+If '--nprocs' is unspecified, a value of 'N=1' is assumed.
+
+The *submit* command enqueues the job and prints its numerical Job ID
+on standard output.
+
+The *run* command does the same interactively, blocking until the job
+has completed.
+
+The intent is for the "mini" commands to remain simple with stable interfaces
+over time, making them suitable for use in scripts.  For advanced usage,
+see flux-run(1) and flux-submit(1).
+
+The available OPTIONS are detailed below.
+
+
+JOB PARAMETERS
+--------------
+These commands accept only the simplest parameters for expressing
+the size of the parallel program and the geometry of its task slots:
+
+*-n, --ntasks=N*::
+Set the number of tasks to launch (default 1).
+
+*-c, --cpus-per-task=N*::
+Set the number of cores to assign to each task (default 1).
+
+*-N, --nodes=N*::
+Set the number of nodes to assign to the job.  Tasks will be distributed
+evenly across the allocated nodes.  It is an error to request more nodes
+than there are tasks.  If unspecified, the minimum number of nodes
```

Comment:
Maybe I'm misinterpreting, but I'm interpreting this statement to mean that the front-end tool will request the minimal number of nodes required to satisfy the user's overall request.  IIUC, if `nodes` is unspecified, the front-end tool omits `nodes` from the jobspec, and the scheduler will make it's decision without constraining the number of nodes in any way (you could get N nodes where N is the number of tasks, or 1 node, just depends on the scheduler).

Reply:
Right, that statement was not only misinformed, but is an inappropriate level of detail for a behavior that is configurable elsewhere.